exasol_advanced_analytics_framework/deployment/scripts_deployer_cli.py

```diff
@@ -10,8 +10,10 @@
 @click.option('--pass', 'pwd', type=str)
 @click.option('--schema', type=str, required=True)
 @click.option('--language-alias', type=str, default="PYTHON3_AAF")
-def scripts_deployer_main(dsn: str, user: str, pwd: str,
-                          schema: str, language_alias: str):
+@click.option('--develop', type=bool, is_flag=True, required=False)
```

Comment:
options by default should not be required, required options usually tend to indicate that the CLI interface could use some rework.